ruff_lsp/server.py

```diff
@@ -821,26 +829,44 @@ async def code_action(params: CodeActionParams) -> list[CodeAction] | None:
     first, and if there's no cell with the given URI, it will fallback to the text
     document.
     """
+
+    def document(uri: str, kind: str) -> Document:
+        if kind in (
+            # For `notebook`-scoped actions, use the Notebook Document instead of
+            # the cell, despite being passed the URI of the first cell.
+            # See: https://github.com/microsoft/vscode/issues/193120
+            f"notebook.{CodeActionKind.SourceFixAll.value}",
+            f"notebook.{CodeActionKind.SourceOrganizeImports.value}",
+            f"notebook.{CodeActionKind.SourceFixAll.value}.ruff",
+            f"notebook.{CodeActionKind.SourceOrganizeImports.value}.ruff",
```

Comment:
Should these be constants defined elsewhere so they can't ever mismatch the ones in `CodeActionOptions`?